src/dodal/devices/areadetector/andor2_epics/andor2_controller.py

```diff
@@ -0,0 +1,79 @@
+import asyncio
+
+from ophyd_async.core import DetectorControl, DetectorTrigger
+from ophyd_async.core._detector import TriggerInfo
+from ophyd_async.epics import adcore
+from ophyd_async.epics.adcore import (
+    DEFAULT_GOOD_STATES,
+    DetectorState,
+    stop_busy_record,
+)
+
+from .andor2_io import (
+    Andor2DriverIO,
+    Andor2TriggerMode,
+    ImageMode,
+)
+
+
+class Andor2Controller(DetectorControl):
+    """
+    Andor 2 controller
+
+    """
```

Comment:
Could: This docstring isn't very informative. I would either expand to give more detail about how the Andor is controlled or remove this.